project-base/storefront/components/Pages/Order/ContactInformation/FormBlocks/ContactInformationEmail.tsx

```diff
@@ -62,16 +60,19 @@ export const ContactInformationEmail: FC = () => {
                 }}
             />
             {isCustomerUserRegisteredData?.isCustomerUserRegistered && !isUserLoggedIn && (
-                <Button
-                    className="mb-5"
-                    size="small"
-                    tid={TIDs.login_in_order_button}
-                    type="button"
-                    onClick={openLoginPopup}
-                >
-                    {t('User with this email is already registered. Do you want to sign in')}
-                </Button>
+                <div className="flex flex-col gap-2 mt-4">
+                    <span>{t('User with this email is already registered')}</span>
+                    <Button
+                        className="w-fit"
+                        size="small"
+                        tid={TIDs.login_in_order_button}
+                        type="button"
+                        onClick={openLoginPopup}
+                    >
+                        {t('Sign in')}
+                    </Button>
+                </div>
             )}
-        </>
+        </FormBlockWrapper>
```

Comment:
Why was this changed? I thought we wanted the entire content as a button

Reply:
It was inspired by breno and agreed by Martin
But the page will be restyling in the future, so maybe Martin will suggest something else 🤔